views/login.hbs

```diff
@@ -0,0 +1,26 @@
+<div class="row">
+  <form class="col s12" action="/login" method="post">
+    {{# if user.errors}}
+      <div class="row">
+        <div id="loginFormError" class="col s12 pink-text">
```

Comment:
move style to stylesheet? maybe chance to error class and have that class be pink-text?

Reply:
meh, I'll remove dup when it happens